openedx/core/djangoapps/discussions/apps.py

```diff
@@ -2,10 +2,31 @@
 Configure the django app
 """
 from django.apps import AppConfig
+from edx_django_utils.plugins import PluginSettings
+from edx_django_utils.plugins import PluginURLs
+
+from openedx.core.djangoapps.plugins.constants import ProjectType
+from openedx.core.djangoapps.plugins.constants import SettingsType
 
 
 class DiscussionsConfig(AppConfig):
     """
     Configure the discussions django app
     """
     name = 'openedx.core.djangoapps.discussions'
+    plugin_app = {
+        PluginURLs.CONFIG: {
+            ProjectType.LMS: {
+            },
+        },
+        PluginSettings.CONFIG: {
+            ProjectType.LMS: {
```

Comment:
Do we need anything here for CMS/studio?

Reply:
Not at this time. Technically we could forgo this too, but I like having some common infrastructure in place.